xrayvision/mem.py

```diff
@@ -584,6 +584,7 @@ def mem(
     maxiter: int = 1000,
     tol: float = 1e-3,
     map: bool = True,
+    total_flux=None,
```

Comment:
```suggestion
    total_flux=Optional[Quantity]
```

I'm not sure of the exact syntax but the docs pick up the types from the type defs no the docstrings you can see the difference here https://xrayvision--78.org.readthedocs.build/en/78/generated/api/xrayvision.mem.mem.html#xrayvision.mem.mem